tests/tests_integration/test_api/test_data_modeling/test_data_models.py

```diff
@@ -24,6 +25,13 @@ def cdf_data_models(cognite_client: CogniteClient) -> DataModelList:
     return data_models
 
 
+@pytest.fixture(scope="function")
+def movie_model(cdf_data_models: DataModelList) -> DataModel:
+    movie_model = next(m for m in cdf_data_models if m.external_id == "Movie")
```

Comment:
`assert` won't be reached as `next` raises `StopIteration` when not found. How about:
```suggestion
    movie_model = cdf_data_models.get(external_id="Movie")
```